include/SFML/Window/Sensor.h

```diff
@@ -0,0 +1,89 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2015 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+#ifndef SFML_SENSOR_H
+#define SFML_SENSOR_H
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Window/Export.h>
+#include <SFML/Window/Types.h>
+#include <SFML/System/Vector3.h>
+
+
+////////////////////////////////////////////////////////////
+/// \brief Sensor Types
+///
+////////////////////////////////////////////////////////////
+typedef enum
+{
+    sfSensorTypeAccelerometer,    ///< Measures the raw acceleration (m/s�)
+    sfSensorTypeGyroscope,        ///< Measures the raw rotation rates (degrees/s)
+    sfSensorTypeMagnetometer,     ///< Measures the ambient magnetic field (micro-teslas)
+    sfSensorTypeGravity,          ///< Measures the direction and intensity of gravity, independent of device acceleration (m/s�)
+    sfSensorTypeUserAcceleration, ///< Measures the direction and intensity of device acceleration, independent of the gravity (m/s�)
+    sfSensorTypeOrientation,      ///< Measures the absolute 3D orientation (degrees)
+
+    sfSensorTypeCount             ///< Keep last -- the total number of sensor types
+} sfSensorType;
```

Comment:
I was looking at how other enums are named in CSFML, and wondered if we could shorten these names somehow, either ```sfSensorAccelerometer``` or even ```sfAccelerometer```.